app/services/classroom_config.rb

```diff
@@ -47,8 +48,65 @@ def configurable?(repo)
   # Returns nothing
   def generate_issues(repo, tree_sha)
     @github_repository.tree_objects(tree_sha).each do |issue|
-      blob = @github_repository.blob(issue.sha)
-      repo.create_issue(blob.data["title"], blob.body) if blob.data.present?
+      blob = formated_blob(@github_repository.blob(issue.sha))
+      next if blob.title.blank?
+      repo.create_issue(blob.title, blob.body, labels: blob.labels.present? ? blob.labels : [])
     end
   end
+
+  # Internal: Generates labels for the assignment_repository based on the configs
+  #
+  # repo - GitHubRepository for which to perform the configuration
+  #                   setups
+  # tree_sha     - sha of the "labels" tree
+  #
+  # Returns nothing
+  def generate_labels(repo, tree_sha)
+    delete_existing_labels(repo)
+    @github_repository.tree_objects(tree_sha).each do |label|
+      blob = formated_blob(@github_repository.blob(label.sha))
+
+      repo.create_label(blob.label, blob.color || "ffffff") if blob.label.present?
```

Comment:
Can we make this string a constant? `DEFAULT_LABEL_COLOR` or something like that, maybe.